Core/Core/Features/Planner/CalendarMain/CalendarViewController.swift

```diff
@@ -234,12 +234,7 @@ class CalendarViewController: ScreenViewTrackableViewController {
 
         // Manually trigger a calendar height update upon rotation
         if traitCollection.verticalSizeClass != previousTraitCollection?.verticalSizeClass {
-            if #available(iOS 17.0, *) {
-                // On iOS 17 embedded VC traits need to be updated first, otherwise the size values from
-                // the embedded VC will be outdated in `updateExpanded()`.
-                // This would be also needed if we used `registerForTraitChanges()`, unfortunately.
```

Comment:
I would leave this explanatory comment in case someone comes around and needs to modify something related to this.